src/hotspot/share/cds/heapShared.cpp

```diff
@@ -647,16 +785,32 @@ void HeapShared::copy_interned_strings() {
 
 void HeapShared::copy_special_subgraph() {
   copy_interned_strings();
+
   init_seen_objects_table();
-  archive_java_mirrors();
-  archive_strings();
-  Universe::archive_exception_instances();
+  {
```

Comment:
Why was this scope introduced?

Reply:
To indicate that these calls must be made while the "see objects" table is available.